_data/repos.json

```diff
@@ -1,532 +1,548 @@
-      "homepage": ""
-    },
-    {
-      "name": "Scap Security Guide (SSG)",
-      "html_url": "https://github.com/ComplianceAsCode/content",
-      "forks_count": 296,
-      "stargazers_count": 676,
-      "description": "Security compliance content in SCAP, Bash, Ansible, and other formats",
-      "homepage": "https://www.open-scap.org/security-policies/scap-security-guide"
-    },
-    {
-      "name": "NB Gallery",
-      "html_url": "https://github.com/nbgallery",
-      "forks_count": 0,
-      "stargazers_count": 0,
-      "description": "nbgallery is a user-friendly Jupyter notebook sharing and collaboration platform.",
-      "homepage": "https://nbgallery.github.io/"
-    },
-    {
-      "name": "Apache NIFI",
-      "html_url": "",
-      "forks_count": 0,
-      "stargazers_count": 0,
-      "description": "Automates the flow of data between systems. NiFi implements concepts of Flow-Based Programming and solves common data flow problems faced by enterprises.",
-      "homepage": "https://nifi.apache.org/"
-    },
-    {
-      "name": "Open Network Operations Platform",
-      "html_url": "https://github.com/onop",
-      "forks_count": 0,
-      "stargazers_count": 0,
-      "description": "Radically simplifies the operation of enterprise networks with SDN applications that reside on top of an OpenFlow-capable network controller.",
-      "homepage": ""
-    },
-    {
-      "name": "RedhawkSDR",
-      "html_url": "https://github.com/redhawksdr",
-      "forks_count": 0,
-      "stargazers_count": 0,
-      "description": "A software-defined radio (SDR) framework designed to support the development, deployment, and management of real-time software radio applications.",
-      "homepage": "https://redhawksdr.github.io/"
-    },
-    {
-      "name": "SECURITY-ENHANCED LINUX (SELINUX)",
-      "html_url": "https://github.com/SELinuxProject",
-      "forks_count": 0,
-      "stargazers_count": 0,
-      "description": "A mandatory access control mechanism in the Linux kernel that checks for allowed operations after standard discretionary access controls are checked. It can enforce rules on files and processes in a Linux system, and on the actions they perform, based on defined policies. SELinux has been part of the Linux kernel since version 2.6.0.",
-      "homepage": "https://selinuxproject.org/page/Main_Page"
-    },
-    {
-      "name": "SECURITY ENHANCEMENTS FOR ANDROID (SEANDROID)",
-      "html_url": "",
-      "forks_count": 0,
-      "stargazers_count": 0,
-      "description": "Confines privileged processes based on security policies by enforcing mandatory access control over all Android processes. SE for Android has been part of Android since Android 4.3.",
-      "homepage": "https://source.android.com/security/selinux"
-    }
+      {
+          "name": "SIMP",
+          "html_url": "https://github.com/NationalSecurityAgency/SIMP",
+          "forks_count": 286,
```

Comment:
There's a lot of whitespace changes to this file and the other json files. Please run the `jq` command to format the json in a standard way.